templates/frontend/objects/article_details.tpl

```diff
@@ -69,8 +69,15 @@
 {/if}
 <article class="obj_article_details">
 
+	{* Indicate if this is only a preview *}
+	{if $article->getData('status') !== $smarty.const.STATUS_PUBLISHED || $publication->getData('status') !== $smarty.const.STATUS_PUBLISHED}
```

Comment:
Only check the publication here. In theory, a preview should work for a second version, where the article is published but the publication is not.

Reply:
Good point, I haven't thought about several publications! Preview button appears only if the submission isn't published and is at least on copyediting stage:
```
<pkp-button v-else-if="submission.status !== getConstant('STATUS_PUBLISHED') && submission.stageId >= getConstant('WORKFLOW_STAGE_ID_EDITING')"
```
Does it make sense to make an additional check here, e.g., if the current publication isn't published, show a preview button that links to that publication (not just to the submission page)?

Reply:
Yes, that'd be great if it's not too much trouble.